manifests/init.pp

```diff
@@ -36,47 +38,51 @@
   $data_connection_timeout = '120',
   $nopriv_user             = undef,
   $async_abor_enable       = 'NO',
+  $accept_timeout          = '60',
+  $one_process_model       = 'NO',
   $ascii_upload_enable     = 'NO',
   $ascii_download_enable   = 'NO',
   $ftpd_banner             = undef,
   $chroot_local_user       = 'NO',
   $chroot_list_enable      = 'NO',
   $chroot_list_file        = '/etc/vsftpd/chroot_list',
-  $ls_recurse_enable       = 'NO',
+  $secure_chroot_dir       = undef,
+  $ls_recurse_enable       = 'YES',
```

Comment:
we should not have changed the default value, instead should specify our value in hiera